src/components/Errors/GlobalErrorsList/index.tsx

```diff
@@ -153,6 +163,26 @@ export const GlobalErrorsList = () => {
     SetGlobalErrorsDataPayload
   >(dataFetcherConfiguration, payload);
 
+  const isDismissalViewModeButtonVisible =
+    data?.dismissedCount && data.dismissedCount > 0; // isUndefined - check for backward compatibility, always show when BE does not return this counter
```

Comment:
Check if dismissedCount exists:
- is not undefined
or
- check it via the feature flag